boa/src/context.rs

```diff
@@ -294,10 +300,21 @@ impl Context {
 
     /// Sets up the default global objects within Global
     #[inline]
-    fn create_intrinsics(&mut self) {
-        let _timer = BoaProfiler::global().start_event("create_intrinsics", "interpreter");
-        // Create intrinsics, add global objects here
-        builtins::init(self);
+    fn create_min_intrinsics(&mut self) {
+        let _timer = BoaProfiler::global().start_event("create_min_intrinsics", "interpreter");
+        // Create minimal intrinsics, enough for the interpreter itself to work, add global objects here
+        builtins::init_min(self);
+    }
+
+    /// Sets up the default global objects within Global
+    #[inline]
+    pub fn create_full_intrinsics(&mut self) {
+        let _timer = BoaProfiler::global().start_event("create_full_intrinsics", "interpreter");
+        // Create minimal intrinsics, enough for the interpreter itself to work, add global objects here
```

Comment:
This comment should mention full intrinsics instead, currently it's a copy of line 305.